src/pages/home/report/ContextMenu/BaseReportActionContextMenu.js

```diff
@@ -49,6 +52,7 @@ class BaseReportActionContextMenu extends React.Component {
                             draftMessage: this.props.draftMessage,
                             selection: this.props.selection,
                         })}
+                        description={contextAction.getDescription ? contextAction.getDescription(this.props.selection, this.props.isSmallScreenWidth) : ''}
```

Comment:
```suggestion
                        description={contextAction.getDescription(this.props.selection, this.props.isSmallScreenWidth)}
```

Now noop (() => {}) the `getDescription` for all other actions.

Reply:
Yeah, I was trying to see if we could use something like `_.result()` or even https://github.com/Expensify/expensify-common/blob/main/lib/Func.jsx#L13 and neither of those are really what would be nice. Ideally it would be something where:
- If it's a function
    - invoke the function with the given params
- if it's not a function
    - return a default value

 I had thought about a no-op, but didn't have too be of an opinion on adding it to all the other actions. The noop does make it a bit cleaner.